lib/eksek.rb

```diff
@@ -9,36 +9,10 @@
 # standard error, etc.
 # Accepts the same options as Process.spawn e.g. :chdir, :env.
 class Eksek
-  ALLOWED_METHODS = %i[
-    exit stdout stderr stdouterr success? success!
-  ].freeze
-
   class << self
-    def respond_to_missing? name, include_private
-      method_name_valid?(name) || super
-    end
-
-    def method_missing symbol, *args, &block
-      return super unless method_name_valid? symbol
-      methods = symbol.to_s.split('_').map(&:to_sym)
+    def run *args, &block
```

Comment:
Consider `def run cmd, opts, &block` so that the manual validation becomes a bit superfluous (yeah, type checking can be considered an anti-pattern where duck-typing is a pattern 🤔)

In that case we could remove the `validate_args` method.

Reply:
`Eksek.run` sounds a bit like "Say.tell" or "Walk.go" or Moon.moon".

Reply:
What if it was something like `Eksek.ute`.

Or `Eksek.cute` 😍

Reply:
Please only mind the first comment in this thread.